packages/beacon-node/src/eth1/eth1MergeBlockTracker.ts

```diff
@@ -96,7 +96,7 @@ export class Eth1MergeBlockTracker {
           // Set latestBlock stats
           metrics.eth1.eth1LatestBlockNumber.set(this.latestEth1Block.number);
           metrics.eth1.eth1LatestBlockTD.set(Number(this.latestEth1Block.totalDifficulty / this.safeTDFactor));
-          metrics.eth1.eth1LatestBlockTimestamp.set(this.latestEth1Block.timestamp);
+          metrics.eth1.eth1LatestBlockTimestamp.set(this.latestEth1Block.timestamp * 1000);
```

Comment:
Prometheus metrics always deal with IS units, so in case of time it must be seconds

Reply:
moved collection in seconds, and update the dashboard expression (*1000) which displays with unit "From now" which seems to be using ms.